app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -901,7 +901,8 @@ private void enterVRVideo(@VideoProjectionMenuWidget.VideoProjectionFlags int aP
         mAttachedWindow.setVisible(false);
 
         closeFloatingMenus();
-        if (aProjection != VideoProjectionMenuWidget.VIDEO_PROJECTION_3D_SIDE_BY_SIDE) {
+        if (aProjection != VideoProjectionMenuWidget.VIDEO_PROJECTION_3D_SIDE_BY_SIDE &&
```

Comment:
I'd suggest refactoring this check in a new private method and use it in the three spots

Reply:
Done